plugins/BEdita/Core/tests/TestCase/Model/Entity/ObjectTypeTest.php

```diff
@@ -190,7 +190,7 @@ public function testGetAlias()
     }
 
     /**
-     * Data provider for `testGetSetPluralized` test case.
+     * Data provider for `testGetSetsingular` test case.
```

Comment:
This is a data provider for `testGetSetTable`. The comment was wrong before, but maybe we can take the chance to fix.